.github/workflows/github-actions.yml

```diff
@@ -1,3 +1,7 @@
+# The purpose of this workflow file is to 
```

Comment:
I feel that there are two things that you have misunderstood a few points here. 
Do correct me if I'm wrong! 

1. It doesn't help to automatically deploy the app. In fact, it helps to ensure that the build passes in case of compilation errors. 
2. Running the linter and testing files is to make sure that they pass. 

Do let me know if there are any issues with respect to this!

Reply:
Oh I see, thank you for pointing out my misunderstanding of the purpose of the workflow file!